src/uu/mkfifo/src/mkfifo.rs

```diff
@@ -88,3 +68,26 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
 
     exit_code
 }
+
+pub fn uu_app() -> App<'static, 'static> {
+    App::new(executable!())
+        .name(NAME)
+        .version(crate_version!())
+        .usage(USAGE)
+        .about(SUMMARY)
+        .arg(
+            Arg::with_name(options::MODE)
+                .short("m")
+                .long(options::MODE)
+                .help("file permissions for the fifo")
+                .default_value("0666")
+                .value_name("0666"),
+        )
+        .arg(
+            Arg::with_name(options::SE_LINUX_SECURITY_CONTEXT)
+                .short(options::SE_LINUX_SECURITY_CONTEXT)
+                .help("set the SELinux security context to default type")
+        )
+        .arg(Arg::with_name(options::CONTEXT).long(options::CONTEXT).value_name("CTX").help("like -Z, or if CTX is specified then set the SELinux\nor SMACK security context to CTX"))
```

Comment:
I'm surprised `rrustfmt` doesn't do anything with this

Reply:
Apparently rustfmt gives up for the whole statement when it encounters a line it can't format because it is too long. Means we should add some line breaks to a string literal to make it shorter. Somewhat related I'd also like to enable the wrap_help feature for clap so that help text is wrapped according to terminal width

Reply:
Sounds good!